build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 	id 'eclipse'
 	id 'idea'
 	id 'maven-publish'
-	id 'fabric-loom' version '0.4.28' apply false
+	id 'fabric-loom' version '0.5.11' apply false
```

Comment:
Im still aware of some issues with loom 0.5 that I need to fix before id be happy to pull this

Reply:
I am aware, but version bump is trivial in regards to everything else.